README.md

```diff
@@ -17,6 +17,10 @@
 ## Overview  
 This web server powers the Bit Mentor Telegram bot. It receives a topic from the bot and dynamically generates questions and answers related to that topic, then sends them to the bot.   
 
+## Server link  
```

Comment:
remove this. we dont need this address to be public, and the users of this repo will not use our deployment, but rather deploy it themselves.
this addr is only for us.
the rest of the world need to know (via the readme) that they need to add env var with the server url, so the bot can send req to it

Reply:
updated the readme to include instructions on what environment variable they need to have